src/components/views/spaces/SpacePanel.tsx

```diff
@@ -344,6 +352,8 @@ const SpacePanel: React.FC = () => {
         }
     });
 
+    const summarizedNotificationState = useGlobalNotificationState();
```

Comment:
Can you move the notifications button into its own component so we're not calling hooks when we don't use their resulting output values, its a bad smell